LocalSettings.php

```diff
@@ -5708,6 +5709,19 @@
 		'fbwikiwiki' => array('visualeditor'),
 		'spiralwiki' => array('visualeditor'),
 	),
+	
+	// Protect site config
+	'wgProtectSiteLimit' => array(
+		'default' => '1 week',
+		'tnoteswiki' => 'indefinite',
+	),
+	'wgProtectSiteDefaultTimeout' => array(
+		'default' => '1 hour',
+		'tnoteswiki' => '2 hours',
+	),
+	'wgProtectSiteExempt' => array(
```

Comment:
It's not a big deal if we do define the default, of course, just don't see why this should be taking up more space